content/posts/newsletter-015/index.md

```diff
@@ -668,6 +668,47 @@ on the module's [repository][Proton-Media-Converter-Github]
 [Proton-Github]: https://github.com/ValveSoftware/Proton
 [Proton-Media-Converter-Github]: https://github.com/ValveSoftware/Proton/tree/proton_5.13/media-converter
 
+### [Rust GPU][RustGPU-Github] v0.1
+
+![Rust GPU Sky](rustgpu.jpg)
+_Sky example in Rust GPU_
+
+Rust GPU is a very early stage project to make Rust a first-class language and
+ecosystem for building GPU code.
+
+[Rust GPU released v0.1 in October][RustGPU-Twitter], and has already garnered
+over 2000 stars on Github. There are also threads about it [on Hacker
+News][RustGPU-HackerNews] and [Reddit][RustGPU-Reddit]. Currently, compiling and
+running very simple shaders works, and a significant portion of the core library
+also compiles. However, many things aren't implemented yet: for example, loops
```

Comment:
...aren't implemented yet: for example, `while`-loops and `if`-statements work, but `for`-loops, iterators, and `match`/`switch` aren't supported yet!

Reply:
:+1: